checker/tests/nullness/java8inference/Issue980.java

```diff
@@ -21,9 +21,9 @@ void m(List<String> strings) {
         @SuppressWarnings("nullness")
```

Comment:
Is this `@SuppressWarnings` unnecessary? As `collectedStrings` no longer produces a warning, I would expect that this one also doesn't.